rocketpy/rocket/rocket.py

```diff
@@ -871,7 +875,9 @@ def add_tail(
         self.add_surfaces(tail, position)
         return tail
 
-    def add_nose(self, length, kind, position, bluffness=0, name="Nose Cone"):
+    def add_nose(
+        self, length, kind, position, bluffness=0, base_radius=None, name="Nose Cone"
+    ):
```

Comment:
I think this is a breaking change. Although a small one, it is a breaking change and perhaps we should at least mention it. 

I'm sorry that I couldn't execute the steps below before commenting, but here are my thoughts:

An application that uses the following code in rocketpy v1.2.0 and above will stop working:
```python
my_rocket.add_nose(1, "von_karman", 0, 0, "Nose Cone")
```
As you can see, the "Nose Cone" will now be interpreted as the 'base_radius' argument instead of 'name'.

If this is really a breaking change (I'm not sure because you used the `or` operand below and I couldn't run it yet), I see two possible solutions:
- Just ignore the breaking change and assume not a lot of users will suffer from it (which is usually not ok)
- Just move the `base_radius` argument to the last position so it comes after the `name` argument

Reply:
I agree with your statement. While the second option of not causing a break change is more appealing, it breaks pattern with similar methods that leave the name as the last argument.

Having said that, I am ok with this pattern break if it means avoiding a breaking change in a hotfix.

Reply:
Fixed here: [c63cb14](https://github.com/RocketPy-Team/RocketPy/pull/558/commits/c63cb148dbf1e66b162af2a23cabd2896d0bc0e6)